api/config/routes.rb

```diff
@@ -133,6 +133,16 @@
       end
     end
 
+    resources :address_book, only: [], param: :address_id do
+      collection do
+        get :mine
+        post :update
+      end
+      member do
+        post :remove
+      end
+    end
```

Comment:
This seems to go out of the way to avoid being the standard rails rest resource. Is there a reason the standard `index`, `create`, and `destroy` were avoided?